eogrow/tasks/features.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, validity_threshold: float):
 
     def __call__(self, array: np.ndarray) -> bool:
         coverage = np.sum(array.astype(np.uint8)) / np.prod(array.shape)
-        return bool(coverage > self.validity_threshold)  # numpy return np.bool_ by default
+        return bool(coverage > self.validity_threshold)  # noqa SIM901 # numpy return np.bool_ by default
```

Comment:
what about 

```python
(coverage > self.validity_threshold).astype(bool)
```

or dropping the `numpy` layer with

```python
coverage = (np.sum(array.astype(np.uint8)) / np.prod(array.shape)).astype(float)
```

Reply:
the first one works :D it's longer than the `bool` version but ultimately shorter as it gets rid of the comment